### PR TITLE
032 update readme

### PR DESCRIPTION
**Pull request by conormcgee480**
https://trello.com/c/G1MHGi5q

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
+
+    $ java -jar target/kainos-recruitment-api-1.0-SNAPSHOT-sources.jar server config.yml
+
+
+To check that your application is running enter url `http://localhost:8080`
+
+### Testing the application
+You can run the tests contained in the project simply by running the following command.
+
+    $ mvn clean test
```
Comment (seaninbreslin) [inline README.md:55]: Should there be anything in the ReadMe about access to the database in db.properties file

Comment (conormcgee480) [reply README.md:55]: @seaninbreslin made some changes containing the database set up now

**Review by seaninbreslin — APPROVED** [review]

**Comment by conormcgee480** [comment]
[US032 - Update README for running and testing (1/2)](https://trello.com/c/G1MHGi5q/28-us032-update-readme-for-running-and-testing-1-2)

File: README.md [hunk 2]
```diff
@@ -1,14 +1,56 @@
...
-Health Check
----
 
-To see your applications health enter url `http://localhost:8081/healthcheck`
+### Using the terminal
+Connect to your MySQL server from the command line using the following command containing your unique host, username and password credentials.
+
+    $ mysql -h `host` -u `user` -p `password`
+
+Create an empty database and use it. You then can import the given database.sql file found in the github repository by typing:
```
Comment (boydstepnew) [inline README.md]: by running, instead of typing

**Review by boydstepnew — APPROVED** [review]

**Pull request merged**